src/keylessBackup/KeylessBackupPhoneCodeInput.tsx

```diff
@@ -67,11 +64,6 @@ function KeylessBackupPhoneCodeInput({
         verificationStatus={verificationStatus}
         setSmsCode={setSmsCode}
         onSuccess={() => {
-          dispatch(
-            keylessBackupStarted({
```

Comment:
this created a race condition where if the Torus network requests failed before this was dispatched, the failure status would get over-written.

it's also not where keyless backup really starts, so I moved it earlier in the flow

Reply:
I think this also requires a unit test update. There should be a test that checks navigation and if this action was dispatched

Reply:
thanks, I think I checked the wrong test file for this check. and the other test failures due to effectful imports hid this one